nip27.ts

```diff
@@ -56,7 +56,7 @@ export function replaceAll(
   content: string,
   replacer: (match: NostrURI) => string
 ): string {
-  return content.replaceAll(regex(), (uri, value) => {
+  return content.replaceAll(regex(), (uri, value: string) => {
```

Comment:
The type of value was `any` due to the type of `replaceAll`, causing the top function overload to match. Making it a string (the correct value based on the regex) makes it match the correct overload.